setup/scaling/index.md

```diff
@@ -0,0 +1,16 @@
+---
+layout: single
+title:  "Scale Spinnaker"
```

Comment:
Perhaps "Running Spinnaker at Scale" or so?

Reply:
Already [changed](https://www.spinnaker.io/setup/productionize/)

Reply:
> Already changed

In relation to "productionize"...for me, this raises a similar question as discussed elsewhere: if we say that the setup flow results in a production-grade installation, having a separate "productionize Spinnaker" section seems odd?

/cc @lwander @nhayes @skim1420

Reply:
What about Lars's previous suggestion, that we change https://www.spinnaker.io/setup/install/ to say "...configurable to be production ready," rather than "...suitable for use in production."?

Reply:
> to say "...configurable to be production ready," rather than "...suitable for use in production."?

Is there any particular reason to introduce the "intermediate state" of "configurable to be production ready"? Would it instead be worth considering simply including the guidance in the section on Redis as a storage provider, where users trying to install Spinnaker would probably be best placed to make the choice?

Reply:
Oh, I see what you're saying. Finally. 
I think there's more to come in the Productionizing section besides just Redis config. Is that correct, @lwander ?

Reply:
Yup, and there's even things in the other setup header like monitoring, authn/z, which some might consider prereqs to production readiness.